app/api/admin/users/[id]/route.ts

```
@@ -0,0 +1,157 @@
+
+      case 'activate':
+        // Reactivate user's API keys
+        await prisma.apiKey.updateMany({
+          where: { userId },
+          data: { isActive: true }
+        })
+        
+        // Restore role to USER if it was VIEWER (suspended)
+        const user = await prisma.user.findUnique({
+          where: { id: userId }
+        })
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)